src/main/java/by/exadel/internship/mapper/TechUserMapper.java

```diff
@@ -0,0 +1,19 @@
+package by.exadel.internship.mapper;
+import by.exadel.internship.dto.user.TechUserDTO;
+import by.exadel.internship.entity.user.TechUser;
+import org.mapstruct.IterableMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Named;
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface TechUserMapper {
```

Comment:
As far as I remember we decided to remove this class.

Reply:
Sorry for my misunderstanding,  I've dropped it. Done